pom.xml

```diff
@@ -79,10 +79,10 @@
         <h2.version>2.1.214</h2.version>
         <jakarta.persistence.version>2.2.3</jakarta.persistence.version>
         <jakarta.servlet.jakarta-servlet-api.version>6.0.0</jakarta.servlet.jakarta-servlet-api.version>
-        <hibernate-orm.version>5.6.14.Final</hibernate-orm.version>
+        <hibernate-orm.version>5.6.15.Final</hibernate-orm.version>
         <hibernate.c3p0.version>${hibernate-orm.version}</hibernate.c3p0.version>
-        <infinispan.version>14.0.7.Final</infinispan.version>
-        <infinispan.protostream.processor.version>4.6.0.Final</infinispan.protostream.processor.version>
+        <infinispan.version>14.0.9.Final</infinispan.version>
```

Comment:
This line causes the failure.

Reply:
@hmlnarik thanks for looking into this. Do you think is ok to move to 14.0.9 if we perform the backport?

Reply:
Yup. The https://github.com/hmlnarik/keycloak/tree/20562-Update-Keycloak-to-Quarkus-2-13-8-Final-main has 14.0.9.